internal/action/main_test.go

```diff
@@ -336,3 +337,88 @@ production:
 		assert.Len(t, v.Items, 1)
 	}
 }
+
+// fuzzyFindWorksapce finds the first workspace matching the passed match string
+func fuzzyFindWorksapce(match string, workspaceList *tfe.WorkspaceList) *tfe.Workspace {
```

Comment:
Not clear why this is necessary, shouldn't you be able to compute the whole workspace name? This would not work for multiple concurrent test runs.